WordPress/src/main/java/org/wordpress/android/ui/mysite/items/listitem/SiteListItemBuilder.kt

```diff
@@ -87,6 +83,23 @@ class SiteListItemBuilder @Inject constructor(
         } else null
     }
 
+    fun buildDomainsItemForJetpackIfAvailable(site: SiteModel, onClick: (ListItemAction) -> Unit): ListItem? {
```

Comment:
1. There's an unused function `buildDomainsItemIfAvailable` in this class. We might want to remove it.
2. Can we add unit tests in `SiteListItemBuilderTest.kt` to test domains menu item availability?

Reply:
Thanks @ashiagr for a quick look.  🙇🏻‍♂️

>  I can see the feature for Jetpack sites in the WordPress app as well

Is the SiteDomainsFeatureConfig enable on WordPress variant as well?  In any case given that `buildDomainsItemIfAvailable` is removed now, it shouldn't happen anymore.

removed `buildDomainsItemIfAvailable`.  It was meant to be used with WordPress variant
updated `SiteListItemBuilderTest`